src/static/js/Changeset.js

```diff
@@ -903,20 +907,24 @@ class TextLinesMutator {
     let removed = '';
     if (this._isCurLineInSplice()) {
       if (this._curCol === 0) {
```

Comment:
What about line markers? If the first character is a line marker and `this._curCol === 1`, shouldn't it be treated the same as if `this._curCol === 0`? Or is that not possible due to invariants? Unfortunately, I don't think it's possible to add an assert here because TextLinesMutator doesn't have access to the attributes.

Reply:
Maybe I don't understand, but the ops that are applied to lines shouldn't care about lineMarker's special meaning in the editor. ie lineMarker is special in the editor, because it's not rendered and it changes the "meaning" of the whole line, but when it comes to Changeset.js whatever attributes apply to the lineMarker are only applied to this single, "real" character. So we should not treat `this._curCol === 1` as `this._curCol === 0`